src/corona_hakab_model/generation/matrix_generator.py

```diff
@@ -21,14 +20,37 @@
 )
 from generation.matrix_consts import MatrixConsts
 from generation.node import Node
+from bsa.parasym import write_parasym, read_parasym
+from bsa.scipy_sparse import read_scipy_sparse
 
 
 class MatrixData:
     __slots__ = ("matrix_type", "matrix", "depth")
 
+    # import/export variables
+    IMPORT_MATRIX_PATH = "../../output/matrix_data"
```

Comment:
This type of path is dependent on where the script is ran from.
use project_structure.py to find the correct output folder path

Reply:
I agree with Mor, more over if you add new constant that is related to the project structure, please add it to project_structure.py

Reply:
תוקן